source/mainnet/smart-contracts/tutorials/counter/counter-contract.rst

```diff
@@ -185,7 +185,7 @@ Initialize it to create your contract instance, so you are ready to invoke the f
 
     concordium-client contract init <YOUR-MODULE-HASH> --sender <YOUR-ADDRESS> --energy 30000 --contract counter --grpc-port 20001
 
-.. image:: images/initialize.png
+.. image:: images/initialize.jpg
```

Comment:
Could you please keep using png since it is pictures of text, for which jpg is not very appropriate since it's lossy.